src/main/java/seedu/duke/command/FindCommand.java

```diff
@@ -7,6 +7,8 @@
 
     public static final String COMMAND_WORD = "find-n";
 
+    public static final String MESSAGE_USAGE = "";
```

Comment:
Maybe can update this to COMMAND_USAGE to be clearer on what this String is for :)